### ISSUE TITLE
Mod not working on 1.18.1

### ISSUE THREAD

**Issue by USBCharger** [RecursiveG/Mc122477Fix#10]
Hey, I'm using this mod on 1.18.1, and I'm still having the issue getting an extra t or / in chat. I'm running the latest version of Zorin OS 16, which is basically a slightly modified take on Ubuntu, and optifine alongside the mod if this helps at all.

**Comment by RecursiveG** [comment]
Does the problem occur less frequently after installing the mod? Or no change at all?
Also, how well can your computer run Minecraft? As low FPS may cause some issues.

**Comment by USBCharger** [comment]
It occurs either slightly less frequently or doesn't really change much. I get around 30/40 FPS on linux, higher on windows, which is dumb, but I've been told Nvidia's drivers for linux are kind of "eh" though the fact that I do have an older graphics card certainly is the major contributing factor.

**Comment by RecursiveG** [comment]
30~40 FPS shouldn't be low enough to trigger the issue.
But still please try [v1.1.0](https://github.com/RecursiveG/Mc122477Fix/releases/tag/v1.1.0) and see if that works for you.

**Comment by USBCharger** [comment]
Hey, sorry for the late reply, I've been busy with some things, I installed the latest version but am still getting the issue.

I looked into my logs, and it appears the mod is failing to load on my system.

17:22:46.700
Loading 3 mods:
	- fabricloader 0.12.12
	- java 17
	- minecraft 1.18.1
	
I removed all other mods and just had that one in the mods folder, and it failed to load, however other mods do load.

*clock ticks by for about 7 minutes*

...Ah. I've been downloading the forge mod...

Digging a little deeper, when I followed the link from here to the Curseforge website, it said forge or fabric was required, however the latest iteration of this mod I can find has only been updated to 1.16. Is there somewhere I can find a working version for fabric 1.18.1?
